packages/react-native-select-pro/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @mobile-reality/react-native-select-pro
 
+## 2.1.5
+
+### Patch Changes
+
+-   [#209](https://github.com/MobileReality/react-native-select-pro/pull/209) [`7b3229c`](https://github.com/MobileReality/react-native-select-pro/commit/7b3229cdf019c66f2cc5dc763a30d746750b469a) Thanks [@kkaplinski](https://github.com/kkaplinski)! - update dependencies
```

Comment:
please fix formatting of this file because it doesn't look good

https://github.com/MobileReality/react-native-select-pro/blob/61d126aa0140b1c082e8567cc45a46bfa8b97957/packages/react-native-select-pro/CHANGELOG.md